### PR TITLE
Add Fedora 31 template upgrade instructions

### PR DESCRIPTION
**Pull request by rocodes**
Instructions based roughly on https://github.com/freedomofpress/securedrop-workstation/issues/544#issuecomment-632193102

Tagging @conorsch for review as per this morning's standup!

### CODE REVIEW

File: docs/includes/getting_support.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+If you are part of the SecureDrop Workstation Pilot and you have questions about this process or about any other aspect of SecureDrop Workstation, please reach out to us.
```
Comment (conorsch) [inline docs/includes/getting_support.rst:1]: Three cheers for reusable includes! :ok_hand:

File: docs/admin/upgrading_to_fedora_31.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+-----------------------------
+
+Once the template is installed, update it using the Qubes updater (**System Tools > Qubes Update**).
+
+Configure VMs to use the new template
+-------------------------------------
+
+VMs not managed by SecureDrop Workstation must be manually configured in order to use the Fedora 31 template.
+
+It is most important to update the ``work`` and ``vault`` VMs to Fedora 31; however, you may also choose to update ``sys-net``, ``sys-usb``, and ``sys-firewall``.
```
[the remark this answers is not in the document]
Comment (rocodes) [reply docs/admin/upgrading_to_fedora_31.rst]: Sounds good; I was basing that on https://github.com/freedomofpress/securedrop-workstation/issues/544#issuecomment-632193102, but I realize that that hierarchy only applies to existing workstation users and have fixed in 252c1e1. Thanks for catching!

File: docs/admin/upgrading_to_fedora_31.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+   sudo qubes-dom0-update qubes-template-fedora-31
+
+You will see some information from the package manager, including a progress bar.
+
+When the download has concluded, you will be prompted to install the package. Type ``y`` to proceed with the installation.
+
+Update the Fedora-31 template
+-----------------------------
+
+Once the template is installed, update it using the Qubes updater (**System Tools > Qubes Update**).
```
[the remark this answers is not in the document]
Comment (rocodes) [reply docs/admin/upgrading_to_fedora_31.rst]: > Recommending slight tweaks to the guiding language. Process-wise, try the template manager tool, and see if you agree it's a better fit to document!

Have recommended using the Qube Manager to make template changes in 252c1e1, as well as addressed all your other comments (thumbs'd up). Thank you for the review!

**Comment by conorsch** [comment]
Rebased to unblock merge

**Pull request merged**